pkg/kube/resources.go

```diff
@@ -133,32 +134,25 @@ type k8sResource struct {
 var podSpecFields = []string{"jobTemplate", "spec", "template"}
 
 // CreateResourceProvider returns a new ResourceProvider object to interact with k8s resources
-func CreateResourceProvider(ctx context.Context, directory, workload string, c conf.Configuration) (*ResourceProvider, error) {
-	if workload != "" {
-		return CreateResourceProviderFromResource(ctx, workload)
+func CreateResourceProvider(ctx context.Context, auditPath, resourceToAudit string, c conf.Configuration) (*ResourceProvider, error) {
```

Comment:
Curious what triggered the refactor here--dependency updates?

Reply:
This was just renaming parameters for me to understand, no refactor - I will roll back to as it was